packages/xrpl/src/models/utils/index.ts

```diff
@@ -1,4 +1,4 @@
-const HEX_REGEX = /^[0-9A-Fa-f]+$/u
+const HEX_REGEX = /^[0-9A-Fa-f]*$/u
```

Comment:
this variable has been used in multiple places across the code base. What is the necessity for this change?

Its intractable to check if this regex is correct, there are too many uses.

Reply:
Everywhere it's used is fine with an empty string.